django/contrib/admin/templates/admin/base.html

```diff
@@ -64,6 +67,15 @@
     {% endblock %}
     {% endif %}
 
+  <div class="main shifted" id="main">
+      {% if not is_popup and is_nav_sidebar_enabled %}
+          {% block nav-sidebar %}
+              {% include "admin/nav_sidebar.html" %}
+          {% endblock %}
+      {% endif %}
+
+    <div class="content">
```

Comment:
Can we fix indentation? and chop empty lines
```html
  <div class="main shifted" id="main">
    {% if not is_popup and is_nav_sidebar_enabled %}
      {% block nav-sidebar %}
        {% include "admin/nav_sidebar.html" %}
      {% endblock %}
    {% endif %}
    <div class="content">
	  ...
    </div>
  </div> 

```

Reply:
I do prefer this indentation, but it wouldn't be consistent with the rest of the file. Admittedly, the rest of the file isn't very internally consistent already.

Reply:
> 2 spaces in HTML files

The problem is that almost all current HTML files have 4 spaces. I'm suggesting to move this to separate task.

Reply:
`Don’t waste time doing unrelated refactoring of existing code to conform to this style.`..., but we should follow this style in new lines.

Reply:
Ok, I had to change a fair bit as there's a new element that stuff is inside, but I've tried not to change anything unnecessary. I've also fixed the indentation in `app_list.html` since that's a new file.